web/src/pages/_components/CreatorStation.svelte

```diff
@@ -10,7 +10,7 @@
 </script>
 
 <a
-  href={$url("/station/:uid", { uid: station.uid })}
```

Comment:
I think the station is a better route name as you are viewing the station, not the stream.
a stream can be played by multiple stations